lib/govspeak.rb

```diff
@@ -53,6 +53,7 @@ def initialize(source, options = {})
       @source = source ? source.dup : ""
 
       @images = options.delete(:images) || []
+      @relaxed_sanitization_elements = options.delete(:relaxed_sanitization_elements) || []
```

Comment:
This seems quite a long argument that's a bit tricky to understand I somewhat wonder if there is a nicer name for it.  It seems a little bit coupled to "relaxed" of sanitize which seems like something a user of this gem wouldn't need to know about.

I'll have a quick ponder for suggestions.

Reply:
perhaps just simply `allowed_elements` ?

Reply:
I had a similar thought when looking at this code, I agree with `allowed_elements`.

Reply:
Thanks both, renamed 👍